app.js

```diff
@@ -85,7 +85,10 @@ sayGoodbyeButton.addEventListener('click', () => {
     const stillHungry = [];
     for (const friend of friends) {
         // > if the friend is not fully satisfied, push
-        // them into the stillHungry array
+        if (friend.satisfaction < 3) {
```

Comment:
This logic is not working because when you click the goodbye button all the friends disappear (not just the full ones). Do you see why?

Reply:
I see it! I'll correct that now.